pkg/ottl/functions.go

```diff
@@ -469,6 +469,12 @@ func (p *Parser[K]) buildArg(argVal value, argType reflect.Type) (any, error) {
 			return nil, err
 		}
 		return StandardPMapGetter[K]{Getter: arg.Get}, nil
+	case strings.HasPrefix(name, "PSliceGetter"):
+		arg, err := p.newGetter(argVal)
+		if err != nil {
+			return nil, err
+		}
+		return StandardPSliceGetter[K]{Getter: arg.Get}, nil
```

Comment:
Adding the new Getter requires some new tests in `functions_test.go` that hit this case